From 49cc8327ac687e928bb0016a1a4c0112c7d29c5f Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Sat, 15 Feb 2025 12:19:15 +0200
Subject: [PATCH] tests: Fix owe_invalid_assoc_resp with mac80211_hwsim STRICT
 validation

mac80211 has started rejecting the association attempt based on some
HT/WMM information missing for the (Re)Association Response frame when
in STRICT validation mode and that broke this test case with
mac80211_hwsim due to the Association Response frame being constructed
without HT and WMM information in the test script. Disable HT for now to
avoid this.

Signed-off-by: Jouni Malinen <j@w1.fi>
---
 tests/hwsim/test_owe.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/hwsim/test_owe.py b/tests/hwsim/test_owe.py
index 987d1d517..891dd011a 100644
--- a/tests/hwsim/test_owe.py
+++ b/tests/hwsim/test_owe.py
@@ -890,6 +890,7 @@ def test_owe_invalid_assoc_resp(dev, apdev):
     """Opportunistic Wireless Encryption - invalid Association Response frame"""
     check_owe_capab(dev[0])
     params = {"ssid": "owe",
+              "ieee80211n": "0",
               "wpa": "2",
               "ieee80211w": "2",
               "wpa_key_mgmt": "OWE",
-- 
2.43.0

